src/rules/unit-blacklist/README.md

```diff
@@ -49,3 +49,48 @@ a { height: 100vmin; }
 ```css
 a { animation: animation-name 5s ease; }
 ```
+
+## Optional Options
+
+### `ignoreProperties: { unit: ["property", "/regex/"] }`
+
+Ignore the specified unites in the specified properties.
+
+Given:
+
+```js
+["px", "vmin"]
+
+ignoreProperties: {
+  "px": [ "font-size", "/^border/" ],
+  "vmin": [ "width" ]  
+}
+```
```

Comment:
This is unclear. Can you please check for precedent in existing documentation and match that format?